content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx

```diff
@@ -34,28 +34,28 @@ To generate and instantiate Prisma Client:
 
 1. You can now [instantiate the Prisma Client](instantiate-prisma-client) in your code:
 
-    <TabbedContent tabs={[<FileWithIcon text="TypeScript" icon="code"/>, <FileWithIcon text="JavaScript" icon="code"/>]}>
-    <tab>
+   <TabbedContent tabs={[<FileWithIcon text="TypeScript" icon="code"/>, <FileWithIcon text="JavaScript" icon="code"/>]}>
+   <tab>
 
-    ```ts
-    import { PrismaClient } from '@prisma/client'
-    const prisma = new PrismaClient()
-    // use `prisma` in your application to read and write data in your DB
-    ```
+   ```ts
+   import { PrismaClient } from '@prisma/client'
+   const prisma = new PrismaClient()
+   // use `prisma` in your application to read and write data in your DB
+   ```
 
-    </tab>
+   </tab>
 
-    <tab>
+   <tab>
 
-    ```js
-    const { PrismaClient } = require('@prisma/client')
-    const prisma = new PrismaClient()
-    // use `prisma` in your application to read and write data in your DB
-    ```
+   ```js
+   const { PrismaClient } = require('@prisma/client')
+   const prisma = new PrismaClient()
+   // use `prisma` in your application to read and write data in your DB
+   ```
 
-    </tab>
+   </tab>
 
-    </TabbedContent>
+   </TabbedContent>
```

Comment:
Are we sure changing this indentation does not break the formatting in the rendered version?

Reply:
This is the prettier config we have, it runs an on commit hook. I can disable it with `--no-verify` but that kind of defeats the point. I guess the config must have changed from the last time someone made changes to this file and committed them.

Reply:
👋 This is fine - if you make sure you double `<TabbedContent>` and friends, Prettier only reduces four spaces to three and leaves it alone after that.

Reply:
And this renders correctly under the list item?

Reply:
Yep, it does.